Include section keystore management tool (#7000)
wazuh/wazuh-documentation: source/installation-guide/wazuh-server/step-by-step.rst

      .. include:: /_templates/installations/wazuh/common/check_wazuh_manager.rst

>   #. Save the Wazuh indexer username and password into the Wazuh keystore using the wazuh-keystore tool: 

Reviewer: Wazuh keystore -> Wazuh manager keystore

Wazuh indexer, Wazuh dashboard, and Filebeat also have their own keystore, we have to name our components properly

Author: Done in [Wazuh components are named appropriately](https://github.com/wazuh/wazuh-documentation/pull/7000/commits/ef178ca855e5c05eca6c5c8705a7e54e2199f3bb) commit.

      .. include:: /_templates/installations/wazuh/common/check_wazuh_manager.rst

>   #. Add the username and password that the Wazuh manager uses to connect to the indexer, use the keystore tool: 

Reviewer 2: ```suggestion
  #. Save the Wazuh indexer username and password into the Wazuh keystore using the wazuh-keystore tool: 
```

    .. code-block:: console

       # /var/ossec/bin/wazuh-keystore -f indexer -k username -v <INDEXER_USERNAME>
       # /var/ossec/bin/wazuh-keystore -f indexer -k password -v <INDEXER_PASSWORD>   

    .. note:: The default <INDEXER_USERNAME> is admin, as the password is randomly generated, run the following command to get <INDEXER_PASSWORD>.
      
      .. code-block:: console

>          # tar -axf wazuh-install-files.tar wazuh-install-files/wazuh-passwords.txt -O | grep -P "\'admin\'" -A 1

Reviewer 2: wazuh-install-files.tar does not exist on step-by-step installation. Step installation use admin:admin as you can see here https://documentation.wazuh.com/current/installation-guide/wazuh-indexer/step-by-step.html#testing-the-cluster-installation.

The note can clarify this admin:admin because is the step-by-step default installation credentials

Author: Done in [Make requested changes](https://github.com/wazuh/wazuh-documentation/pull/7000/commits/166a6616aa43bcfd53cffd26dd40a3764a21f901) commit
